rules/github-scrum-workflow/rule.md

```diff
@@ -0,0 +1,86 @@
+---
+type: rule
+archivedreason:
+title: How to set up GitHub Scrum workflow
+guid: e63da194-c5d2-4830-9106-d164aafcddb5
+uri: do-you-know-the-how-to-set-up-github-workflow
+created: 2021-06-21T06:44:34.0000000Z
+authors:
+  - title: Tom Bui
+    url: https://www.ssw.com.au/people/tom-bui
+related: []
+
+---
+### How to set up GitHub Scrum workflow
+
+There are several ways of turning your GitHub repository into a simple but powerful agile work environment, the following is the recommended:
+
+<!--endintro-->
+
+Based on [GitHub scrum workflow](https://github.com/jvandemo/github-scrum-workflow)
+
+#### How it works
+
+- Product Backlog Items (PBIs) are reported as **issues**
+- Points and meta data are assigned to items as **labels**
+- **milestones** are used to group issues in Sprints
+
+#### 1. Create issues as backlog items
+
+To create a new backlog item, it is best practice to configure GitHub issue templates for your repository 
+
+Issue templates configuration guide can be found at [Configuring issue templates for your repository](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository)
+
+Assign the issue the right labels and/or assign it to a Sprint (milestone).
+
+Issues allow you to have a conversation about the item and even allow you to create task lists inside the issue using [GitHub's markdown](https://guides.github.com/features/mastering-markdown/).
+
+#### 2. Add labels to issues
+
+Add the following labels to your repository:
+
+### Estimates
+
+`estimate` labels allow you to set estimates in your backlog:
+e.g.:
+| Label        | Time Estimate |
+| -------------|:-------------:|
```

Comment:
Can you double check that the labels align with the rule

Reply:
Hi @wicksipedia 

> Can you double check that the labels align with the rule

Which rule are you referring to?

Reply:
Done - Fixed the label and estimates to align with the rule